modules/installation-vsphere-infrastructure.adoc

```diff
@@ -45,39 +45,43 @@ endif::[]
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 7 instance that meets the requirements for the components that you use.
+You must install the {product-title} cluster on a VMware vSphere version 7.0 Update 2 or later instance that meets the requirements for the components that you use.
 
+[NOTE]
+====
+Version 4.13 of {product-title} supports VMware vSphere version 8.0.
+====
 
 .Version requirements for vSphere virtual environments
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
 |VMware virtual hardware | 15 or later
-|vSphere ESXi hosts | 7.0.2 or later
-|vCenter host   | 7.0.2 or later
+|vSphere ESXi hosts | 7.0 Update 2 or later
+|vCenter host   | 7.0 Update 2 or later
 |===
 
 [IMPORTANT]
 ====
-Installing a cluster on VMware vSphere versions 7.0.0 and 7.0.1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere virtual machines, see the "Updating hardware on nodes running in vSphere" article in the _Updating clusters_ section.
+Installing a cluster on VMware vSphere versions 7.0 and 7.0 Update 1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported. Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later. To update the hardware version for your vSphere virtual machines, see the "Updating hardware on nodes running in vSphere" article in the _Updating clusters_ section.
```

Comment:
> Installing a cluster on VMware vSphere versions 7.0 and 7.0 Update 1 is deprecated. These versions are still fully supported, but all vSphere 6.x versions are no longer supported.

I understand we don't support installation on 7.0.1 in 4.13, right?



> Version 4.12 of {product-title} requires VMware virtual hardware version 15 or later

And we should mention 4.13 here.

cc @sgaoshang

Reply:
Thanks, @duanwei33 . I updated the content to state 4.13.

I'll await @sgaoshang response on dropping the 7.0 and 7.0 Update 1 sentence.

Reply:
I'm also prefer we drop the 7.0 and 7.0 Update 1 sentence, thanks.

Reply:
the 4.13 installer will not allow below vSphere 7 U2 components...that includes *all* esxi hosts in the defined clusters and vcenter.

Reply:
Thank you, all. I removed this admonition from the text.